items.json

```diff
@@ -76599,10 +76599,10 @@
     "name": "Long Fine Steel Spear +3",
     "culture": "Empire",
     "type": "Polearm",
-    "price": 5290,
-    "weight": 1.97,
+    "price": 11701,
+    "weight": 1.79,
     "heirloomLevel": 3,
-    "tier": 5.73965549,
+    "tier": 9.049611,
     "requirement": 0,
     "flags": [],

```

Comment:
Try adding one stat. I'd rather have it tier 10.1 (going slightly over tier 10) then losing 0.7 tier